style.css

```diff
@@ -366,3 +405,149 @@ textarea {
   justify-content: center;
   border-radius: 5px;
 }
+
+@media all and (max-width: 767px) {
+  .nav-link {
+    display: none;
+  }
+
+  .header {
+    max-width: 300px;
+  }
+}
+
+@media all and (min-width: 768px) {
+  header {
+    color: white;
+    background: no-repeat url(images/headerbg.svg);
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .header,
+  .header-content,
+  .header-footer,
+  .social {
+    padding-left: 260px;
+  }
+
+  .header {
+    max-width: 630px;
+  }
+
+  .header-content {
+    max-width: 920px;
+  }
+
+  .my-logo {
+    padding-left: 145px;
+  }
+
+  .container-main {
+    margin: 0;
+    padding-bottom: 10rem;
+  }
+
+  .social-icons {
+    padding-right: 25rem;
+  }
+
+  .card {
+    width: 65%;
+  }
+
+  .card-content-1 h3 {
+    margin-bottom: 0;
+  }
+
+  .card-container-1 {
+    display: flex;
+    flex-direction: row-reverse;
+    column-gap: 1.5rem;
+  }
+
+  .card-content-1 {
+    display: flex;
+    flex-direction: column;
+    justify-self: start;
+    margin-left: 0;
+    padding-left: 2rem;
+    align-items: flex-start;
+    margin-right: auto;
+    padding-top: 2rem;
+  }
+
+  .card-container-1 img {
+    padding-right: 2rem;
+    padding-top: 3rem;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    column-gap: 4rem;
+    padding-top: 2rem;
+  }
+
+  .card-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    column-gap: 2rem;
+    padding-left: 1rem;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: row;
+    column-gap: 3rem;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] You have implemented the about and contact sections design in a good way. Let's make it better by increasing the margin sizes of the top and the bottom of each section.

- Your design:
<img width="646" alt="image" src="https://user-images.githubusercontent.com/77190203/171411355-1724738a-44fb-498c-ba23-2792d7263b5b.png">